src/main/java/org/spongepowered/api/entity/living/Sittable.java

```diff
@@ -42,5 +42,4 @@
      * @param sitting Whether this entity is sitting
      */
     void setSitting(boolean sitting);
-
```

Comment:
It looks like whoever transposed that document from [the original](https://docs.google.com/document/d/1afZMMsU9yODX6d6WZAiKnabMbkuDXa_3ulsuDKZcfxg/edit#heading=h.kfel7xixr2pj) didn't include the statement that that rule is at the author's discretion (which is in agreement with the google style document on that rule). The page should probably be updated to reflect that.